BUILD.md

```diff
@@ -142,6 +142,23 @@ bin/linkerd -n emojivoto stat deployments
 bin/linkerd -n emojivoto tap deploy voting
 ```
 
+#### Deploying Control Plane components with Tracing
```

Comment:
@grampelberg Any thoughts on if BUILD.md is the right place for this? Feels like it should be in `website`.

Reply:
For now, it should be in `BUILD.md` as this is for folks actively doing development. Once we've got a more general demo tracing control plane solution, it should move over to the website.